packages/web-twig/src/Resources/components/Tooltip/README.md

```diff
@@ -54,16 +54,30 @@ yourself or you can use the prepared TooltipWrapper component:
 Please consult the [CSS implementation of Tooltip][tooltip] to help you pick the
 best positioning approach for your use case.
 
+## Advanced floating functionality
+
+To enable the advanced floating functionality, you need to have activated feature flag `spirit-feature-tooltip-enable-data-placement` on any parent element.
+This requirement will be removed in future major version.
+
+Advanced floating functionality is provided by JavaScript plugin and by [Floating UI][floating-ui] library.
+
 ## API
 
 ### Tooltip
 
-| Name            | Type                                         | Default  | Required | Description              |
-| --------------- | -------------------------------------------- | -------- | -------- | ------------------------ |
-| `closeLabel`    | `string`                                     | `Close`  | ✕        | Close label              |
-| `id`            | `string`                                     | `null`   | ✕        | Optional tooltip ID      |
-| `isDismissible` | `bool`                                       | `false`  | ✕        | Make tooltip dismissible |
-| `placement`     | [Placement Dictionary][dictionary-placement] | `bottom` | ✕        | Tooltip placement        |
+| Name                            | Type                                         | Default  | Required | Description                                                                                                                                                                                                                                                                |
+| ------------------------------- | -------------------------------------------- | -------- | -------- | -------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- |
+| `closeLabel`                    | `string`                                     | `Close`  | ✕        | Close label                                                                                                                                                                                                                                                                |
+| `enableControlledPlacement`     | `bool`                                       | false    | ✕        | Enables ability to control Tooltip via external JavaScript like Tooltip JS plugin or custom script.                                                                                                                                                                        |
```

Comment:
I have added this prop to enable all other props for Tooltip plugin, e.g. defaults will not be rendered to the DOM until this prop is used.

Reply:
This will be in the TooltipModern only, I think.

Reply:
I have rewrote the documentation and added the copy about the feature flag.